test/Streamly/Test/FileSystem/Handle.hs

```diff
@@ -0,0 +1,149 @@
+-- |
+-- Module      : Streamly.Test.FileSystem.Handle
+-- Copyright   : (c) 2019 Composewell technologies
+-- License     : BSD-3-Clause
+-- Maintainer  : streamly@composewell.com
+-- Stability   : experimental
+-- Portability : GHC
+
+module Streamly.Test.FileSystem.Handle (main) where
+
+import Data.Word (Word8)
+import Test.Hspec as H
+import Test.Hspec.QuickCheck
+import Test.QuickCheck (Property, forAll, Gen, vectorOf, choose)
+import Test.QuickCheck.Monadic (monadicIO, assert, run)
+import System.IO
+    ( Handle
+    , IOMode(..)
+    , SeekMode(..)
+    , hClose
+    , hFlush
+    , hSeek
+    , openFile
+    )
+import System.IO.Temp (writeSystemTempFile)
+import Streamly.FileSystem.Handle as H
```

Comment:
import `qualified`, as FH.

In new code I prefer using full names for imports e.g. `as Handle`, `as Fold`, `as Stream`, `as Array`, `as Unicode` etc.